.github/workflows/cd.yml

```diff
@@ -65,15 +65,11 @@ jobs:
           echo "MONGO_SERVER=$MONGO_SERVER" >> .env
           echo "MONGO_USERNAME=$MONGO_USERNAME" >> .env
           echo "MONGO_PASSWORD=$MONGO_PASSWORD" >> .env
-          echo "API_KEY=$API_KEY" >> .env
-          echo "API_URL=$API_URL" >> .env
         env:
           # node creds
           MONGO_SERVER: mongo
           MONGO_USERNAME: ${{ secrets.MONGO_USERNAME }}
           MONGO_PASSWORD: ${{ secrets.MONGO_PASSWORD }}
-          API_KEY: ${{ secrets.API_KEY }}
```

Comment:
please revert changes for `ci.yml` and `cd.yml` The ones in latest main branch should be the truth